Util/NetCDF/nc_get_var.m

```diff
@@ -0,0 +1,35 @@
+function [var] = nc_get_var(id,varname,varargin)
```

Comment:
why not using the matlab function 
varData  = ncread('example.nc','varname')

Reply:
because ncread is too high-level and do not support groups. This does.